start.sh

```diff
@@ -3,6 +3,8 @@
 # Chown volumes so that nginx can read/write
 chown www-data:www-data /usr/share/nginx/html/images
 chown www-data:www-data /usr/share/nginx/html/LocalSettings.php
+chown -R www-data:www-data /tmp/extensions/
+cp -r /tmp/extensions/* /usr/share/nginx/html/extensions/
```

Comment:
Ooups, just notice the answer is already in README:

"To help maintain immutability of the container, you may extend this image to include your LocalSettings.php into your own version of MediaWiki. Here is a sample Dockerfile for that

FROM simplyintricate/mediawiki

ADD LocalSettings.php /usr/share/nginx/html/LocalSettings.php"